src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/DetailsViewModelMapper.cs

```diff
@@ -132,5 +136,23 @@ private void SetFundingBandCap(string courseCode, IEnumerable<CohortDraftApprent
 
             return null;
         }
+
+        private string SetFundingBandExcessHeader(int apprenticeshipsOverCap)
```

Comment:
Just a personal preference, instead of Set... can it be Get.... . As we are getting a value.